### DISCUSSION TITLE
Windows Spotlight removed from Settings app after tweak, how to restore it?

### DISCUSSION THREAD

**Discussion by ThePooBurner** [ChrisTitusTech/winutil#2671] [General]
So i tried to roll back the disable cloud tweaks because it removed Windows Spotlight entirely from Personalization in the Settings app.  I actually quite liked the daily wallpaper changes on the login/lock screens.  I've been trying to undo whatever the tweak did for months and i have been entirely unsuccessful. I've looked at every registry key I could find having to do with Spotlight and nothing i change brings it back.  I tried doing a rollback within the utility and it didn't restore any of the settings to defaults or what they were before I applied the tweak.   Can anyone point me in the right direction to get Spotlight available in Settings and working again?

**Comment by vishnurn46** [comment]
Same issue for me as well

**Comment by FabricioFodi** [comment]
I had the exact same issue, Spotlight got stuck on the same lock screen image for months, and after trying to re-enable it, it would either revert to "Picture" or keep loading forever.

What fixed it for me was:

Get Installed Tweaks
<img width="1045" height="131" alt="Captura de tela 2026-07-31 090843" src="https://github.com/user-attachments/assets/5e1a79b4-bef3-4d63-a3f2-a0d253867ad9" />
Undo Selected Tweaks
<img width="422" height="110" alt="Captura de tela 2026-07-31 090916" src="https://github.com/user-attachments/assets/2d27c0e0-6f15-4b00-a978-0a02960b18bc" />

Reboot Windows
Re-enable Windows Spotlight

After that, Spotlight immediately started downloading new lock screen images again.
